apiDI/src/main/java/com/panhuk/apidi/base/BaseApiModule.kt

```diff
@@ -4,20 +4,27 @@ import com.panhuk.api.BuildConfig
 import dagger.Module
 import dagger.Provides
 import okhttp3.OkHttpClient
+import okhttp3.logging.HttpLoggingInterceptor
 import retrofit2.Converter
 import retrofit2.Retrofit
 import retrofit2.converter.gson.GsonConverterFactory
 
 @Module
 class BaseApiModule {
   @Provides
-  fun provideOkhttpClient(): OkHttpClient =
+  fun provideOkhttpClient(loggingInterceptor: HttpLoggingInterceptor): OkHttpClient =
     OkHttpClient.Builder()
+      .addInterceptor(loggingInterceptor)
       .build()
 
   @Provides
   fun provideConverterFactory(): Converter.Factory = GsonConverterFactory.create()
 
+  @Provides
+  fun createLoggingInterceptor(): HttpLoggingInterceptor {
```

Comment:
not `create` but `provide`, follow the same style

Reply:
dublicate probably because of merge conflict